### DISCUSSION TITLE
Fuse js implementation not working initially

### DISCUSSION THREAD

**Discussion by studiosd** [jpanther/congo#479] [Q&A] [answered]
Thank you for the great theme. I am using a heavily adjusted version, which works great. One thing that does not work is the Fuse search, which shows the parameters as in the attached image when a first search is conducted, and returns no results. All subsequent searches work fine. I have not touched Fuse Js, which is why I am posting this question in the Discussion here. Any help would be greatly appreciated as I have no idea how to go about this any more and would otherwise replace Fuse Js with another search tool.
![Screenshot 2023-02-21 at 18 08 37](https://user-images.githubusercontent.com/126004845/220413993-dbe35dc2-76cd-4401-8b7a-6e7dfb4aa17b.jpg)

**Comment by stereobooster** [comment, marked as answer]
Looks like this interpolation didn't work

```js
    results.forEach(function (value, key) {
      resultsHTML =
        resultsHTML +
        `<li class="mb-2">
          <a class="flex items-center px-3 py-2 rounded-md appearance-none bg-neutral-100 dark:bg-neutral-700 focus:bg-primary-100 hover:bg-primary-100 dark:hover:bg-primary-900 dark:focus:bg-primary-900 focus:outline-dotted focus:outline-transparent focus:outline-2" href="${value.item.permalink}" tabindex="0">
            <div class="grow">
              <div class="-mb-1 text-lg font-bold">${value.item.title}</div>
              <div class="text-sm text-neutral-500 dark:text-neutral-400">${value.item.section}${value.item.date == null ? '' : `<span class="px-2 text-primary-500">&middot;</span>${value.item.date}</span>`}</div>
              <div class="text-sm italic">${value.item.summary}</div>
            </div>
            <div class="ml-2 ltr:block rtl:hidden text-neutral-500">&rarr;</div>
            <div class="mr-2 ltr:hidden rtl:block text-neutral-500">&larr;</div>
          </a>
        </li>`;
    });
```

Any chance that you replaced back tick (`) with quote (')?

**Comment by studiosd** [comment]
Thanks stereobooster! I had a different typo in the section you pointed out, so that solved my problem. Thank you especially for taking a look at such an old question.
